client/src/components/Explorer/ExplorerToggle.tsx

```diff
@@ -7,10 +8,15 @@ import * as actions from './actions';
 import Button from '../../components/Button/Button';
 import Icon from '../../components/Icon/Icon';
 
+interface ExplorerToggleProps {
+  onToggle(): void;
+  children: any[];
```

Comment:
Should we add more specifics to this type?
maybe `children: React.ReactNode`
While we may pass children in as an array here, there is no reason it should not compile if in the future we pass in a single child.

Note: from what I can see this `children` is a react element rendered between the icon menu items and not the children (as in page children data).

Reply:
Yep, I'll fix this up!